plugins/Live/templates/_dataTableViz_visitorLog.twig

```diff
@@ -190,6 +190,7 @@ GPS (lat/long): {{ visitor.getColumn('latitude') }},{{ visitor.getColumn('longit
             {% endif %}
 
             <div class="col-md-{% if displayVisitorsInOwnColumn %}7{% else %}8{% endif %} column {% if visitor.getColumn('visitConverted') and not isWidget %}highlightField{% endif %}">
+                {% block beforeActionsInfo %}{% endblock %}
```

Comment:
We want to expose as less as possible as API. The events is a solution that will allow us easier to maintain it and to not break API. For this reason we cannot let anyone change view variables etc. Instead, for now, the solution is to use events unless we have possibly more stuff in Angular etc. 

The ideal solution would be to describe what you need to add and we could see if we can provide a stable API that let's any plugin do this. This way it would be completely unrelated to templates etc. But it's not always possible.